source/adapters/level_zero/v2/queue_immediate_in_order.cpp

```diff
@@ -11,8 +11,51 @@
 #include "queue_immediate_in_order.hpp"
 
 namespace v2 {
+
+static int32_t getZeOrdinal(ur_device_handle_t Device, queue_group_type type) {
+  if (type == queue_group_type::MainCopy && Device->hasMainCopyEngine()) {
+    return Device->QueueGroup[queue_group_type::MainCopy].ZeOrdinal;
+  }
+  return Device->QueueGroup[queue_group_type::Compute].ZeOrdinal;
+}
+
+static std::optional<int32_t> getZeIndex(const ur_queue_properties_t *Props) {
+  if (Props) {
+    if (Props->pNext) {
```

Comment:
This would be more readable as `if (Props && Props->pNext)`

Reply:
Sure, done.